app/ui-react/packages/api/src/useExtensions.tsx

```diff
@@ -8,13 +8,28 @@ export interface IExtensionsResponse {
   totalCount: number;
 }
 
-export const useExtensions = (disableUpdates: boolean = false) => {
+type IExtensionType = "Steps" | "Connectors" | "Libraries";
+
+export const useExtensions = (disableUpdates: boolean = false, extensionType?: IExtensionType) => {
+  /**
+   * Here we are allowing calls to `useExtensions` to pass an optional `extensionType`
+   * query param of type IExtensionType, which gets appended to the string URL when
+   * provided.
+   *
+   * We currently use this for listing available Integration Extensions in the final
+   * step of the Integration Editor.
+   */
+  let extensionUrl: string = `/extensions?per_page=50`;
+  if (extensionType) {
+    extensionUrl = extensionUrl + `&extensionType=${extensionType}`
```

Comment:
instead I'd suggest:

```suggestion
    extensionUrl = `${extensionUrl}&extensionType=${extensionType}`;
```